openpype/settings/entities/schemas/projects_schema/schema_project_fusion.json

```diff
@@ -45,6 +45,29 @@
                     ]
                 }
             ]
+        },
+        {
+            "type": "dict",
+            "key": "copy_fusion_settings",
+            "collapsible": true,
+            "label": "Local Fusion profile settings",
+            "children": [
+                {
+                    "key": "copy_path",
+                    "type": "path",
+                    "label": "Local Fusion profile directory"
+                },
+                {
+                    "type": "boolean",
+                    "key": "copy_status",
+                    "label": "Copy profile on first launch"
+                },
+                {
+                    "key":"force_sync",
+                    "type": "boolean",
+                    "label": "Resync profile on each launch"
```

Comment:
What happens if you were to enable always resync this but disable the option above? Would it still always sync?

Would this be easier as an enum setting `copy_local_prefs` or sync local prêts which has options "never", "first launch" or "always"?

And should first launch be more descriptive? It's not really only first launch. It's only if the destination settings folder does not exist. Can we come up with a short label that describes that better?